crates/ollama-rs/src/generation/chat/mod.rs

```diff
@@ -199,14 +199,13 @@ impl Ollama {
     /// Without impact for existing history
     /// Used to prepare history for request
     fn get_chat_messages_by_id(&mut self, history_id: impl ToString) -> Vec<ChatMessage> {
+        let mut new_history = std::sync::Arc::new(std::sync::RwLock::new(MessagesHistory::default()));
```

Comment:
why this change?

Reply:
Cargo build not successful for me until I do this change. It throws an Error: "creates a temporary value which is freed while still in use".

Reply:
rustc --version?

Reply:
My Rust version was old. Restored.